sites.json

```diff
@@ -1827,7 +1827,18 @@
             "flixster.com"
         ]
     },
-
+    
+    {
+        "name": "Foap",
+        "url": "https://www.foap.com/pages/contact",
+        "difficulty": "medium",
```

Comment:
[Per the contribution guidelines](https://github.com/jdm-contrib/justdelete.me/blob/master/CONTRIBUTING.md) contacting customer support warrants a "hard" difficulty.